### PR TITLE
✨ use Lolliplot relay container in SSMPage

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

**Comment by alex-wilmer** [comment]
I'm getting a mismatched variables error here.. I think it's because the viewer query will run regardless of the `fetchGene` variable? not sure

File: modules/node_modules/@ncigdc/components/SSM.js [hunk 1]
```diff
@@ -289,19 +271,16 @@ const Mutation = compose(
...
-            transcripts={gene.transcripts.hits.edges.map(x => x.node)}
-            selectTranscript={setActiveTranscript}
-            data={lolliplotData}
-          />
-        </Column>
-      )}
+      <Column style={{ ...styles.card, marginTop: '2rem' }}>
+        <Lolliplot
+          mutationId={mutation.ssm_id}
+          geneId={canonicalGeneId}
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/SSM.js]: So this is the part causing the relay error. Looks like we can't have props that are the same name as relay variables.

Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/components/SSM.js]: oh doh >< yeah I knew that but didn't notice it here. thanks!

**Comment by Jephuff** [comment]
@alex-wilmer I linted some as well... want to take  a look?

**Pull request merged**